client/src/main/java/org/apache/rocketmq/client/consumer/LitePullConsumer.java

```diff
@@ -170,6 +187,12 @@ public interface LitePullConsumer {
      */
     void commitSync();
 
+    /**
+     * Offset specified by batch commit
+     * @param messageQueues
+     * @param persist
+     */
+    void commitSync(Map<MessageQueue, Long> messageQueues, boolean persist);
```

Comment:
The first para seems offsets map, not messageQueues.

Reply:
done